lstmcpipe/io/data_management.py

```diff
@@ -67,40 +68,52 @@ def query_continue(question, default="no"):
         return answer
 
 
-def check_data_path(data_path):
+def check_data_path(data_path, glob=None, rglob=None):
```

Comment:
isn't `glob` and `rglob` completely redundant??
one could pass `glob='**/desired_glob'` instead of rglob, no?